docs/pbc/all/dynamic-multistore/202404.0/base-shop/dynamic-multistore-feature-overview.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Dynamic Multistore
 description: Dynamic Multistore lets you create and manage multiple online stores from the Back Office.
-last_updated: July 31, 2023
```

Comment:
Could you please point me to the documentation adjustments that solves the following item from the Jira ticket  (https://spryker.atlassian.net/browse/FRW-7741) description:

1. 

> prepare answers to questions from support in https://docs.google.com/presentation/d/12YlHQg573xt2oo_yWcUlaBaR6tO5g7tb6EKT0q2B3U4/edit#slide=id.g2ca0cf142b9_0_0Connect your Google account and document them if needed and to these

> Issues where we applying runbooks [Recreate Scheduler/Jenkins](https://spryker.atlassian.net/wiki/spaces/CLOUD/pages/3715465852/R55.+Recreate+Scheduler+Jenkins)  these steps no longer works, because the stores are missing in the deploy files

> Issues with Customer PAAS instability because of DMS example [00055936](https://spryker.lightning.force.com/lightning/r/Case/500Se000003e6oCIAQ/view) OPTIBELT | URGENT | New URLs for Stage and Prod where the customer enabled Dynamic Multi Store, without giving us the heads up. Thereby our PAAS configuration for their environments have not yet been applied - giving rise to [00058899](https://spryker.lightning.force.com/lightning/r/Case/500Se000008nA1lIAE/view) The import process will be cancelled during an open connection

> In general, it seems our features that we are releasing lack clarity, on both our internal and customer sides Example: Still unclear about ENV_VARS like SPRYKER_ACTIVE_STORES = "" with DMS this is empty?!

> Big issue for me is the speed and simplicity of on-boarding new engineers to our ever increasing complexity and changes

2. FAQ section is prepared

Reply:
`Support questions are answered here https://spryker.atlassian.net/browse/FRW-8880. `

Issues with Customer PAAS instability because of DMS example [00055936](https://spryker.lightning.force.com/lightning/r/Case/500Se000003e6oCIAQ/view) OPTIBELT | URGENT | New URLs for Stage and Prod where the customer enabled Dynamic Multi Store, without giving us the heads up. Thereby our PAAS configuration for their environments have not yet been applied - giving rise to [00058899](https://spryker.lightning.force.com/lightning/r/Case/500Se000008nA1lIAE/view) The import process will be cancelled during an open connection
**`Incident issues are not related to DMS topic`**